modular_skyrat/modules/moretraitoritems/code/syndiemirror.dm

```diff
@@ -0,0 +1,59 @@
+/obj/item/hhmirror/syndie
```

Comment:
god that typepath is ugly. Not your fault, but man it's so ugly and absolutely not how we do typepath names eugh